apps/chat-e2e/src/testData/conversationHistory/conversationData.ts

```diff
@@ -351,8 +351,18 @@ export class ConversationData extends FolderData {
     return conversation;
   }
 
-  public prepareNestedFolder(nestedLevel: number) {
-    return super.prepareNestedFolder(nestedLevel, FolderType.Chat);
+  // public prepareNestedFolder(nestedLevel: number) {
+  //   return super.prepareNestedFolder(nestedLevel, FolderType.Chat);
+  // }
+  public prepareNestedFolder(
+    nestedLevel: number,
+    folderNames?: Record<number, string>,
+  ) {
+    return super.prepareNestedFolder(
+      nestedLevel,
+      FolderType.Prompt,
```

Comment:
`FolderType.Chat` should be used here. There is a separate class `promptData.ts` to build prompt folders

Reply:
Done